skills/cmux-custom-sidebar/SKILL.md

```
@@ -0,0 +1,92 @@
+This skill is the workflow summary. The complete authoring contract (every supported view, modifier, language feature, and data field) is one command away; read it before writing a non-trivial sidebar:
+
+```bash
+cmux docs sidebars
+curl -fsSL https://raw.githubusercontent.com/manaflow-ai/cmux/main/docs/custom-sidebars.md
+```
+
+## Workflow
+
+1. **Enable the beta** (once). Custom sidebars are behind Settings → Beta features → Custom sidebars (`customSidebars.beta.enabled`). If a written sidebar does not appear in the picker, this flag is the first thing to check.
+2. **Write a named file.** The name becomes the menu label; use short kebab-case:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)